modules/apps/headless/headless-foundation/headless-foundation-test/src/testIntegration/java/com/liferay/headless/foundation/resource/v1_0/test/VocabularyResourceTest.java

```diff
@@ -102,4 +113,42 @@ protected Vocabulary testPutVocabulary_addVocabulary() throws Exception {
 			testGroup.getGroupId(), randomVocabulary());
 	}
 
+	private boolean _equals(AssetType[] assetTypes1, AssetType[] assetTypes2) {
+		if (assetTypes1 == assetTypes2) {
+			return true;
+		}
+
+		if ((assetTypes1 == null) || (assetTypes2 == null)) {
+			return false;
+		}
+
+		if (assetTypes1.length != assetTypes1.length) {
+			return false;
+		}
+
+		for (int i = 0; i < assetTypes1.length; i++) {
+			AssetType assetType1 = assetTypes1[i];
+			AssetType assetType2 = assetTypes2[i];
+
+			if (assetType1 == assetType2) {
+				continue;
+			}
+
+			if ((assetType1 == null) || (assetType2 == null)) {
+				return false;
+			}
+
+			if (Objects.equals(
+					assetType1.getRequired(), assetType2.getRequired()) &&
+				Objects.equals(assetType1.getType(), assetType2.getType()) &&
+				Objects.equals(
+					assetType1.getSubtype(), assetType2.getSubtype())) {
+
+				continue;
+			}
```

Comment:
Aren't we missing an `else {return false;}` here?

Reply:
Yeahhh :)